### PR TITLE
Fix the main image on the spec.

### LINKED ISSUE #9842: [DOCS] Update format specs to use the mkdocs material markdown
#9779 changed to symlinks of formats that are still using the hugo markdown and it broke the links.

### PR DESCRIPTION
**Pull request by bitsondatadev**
Fix the main image on the spec.

### CODE REVIEW

File: format/spec.md [hunk 1]
```diff
@@ -791,7 +788,7 @@ Each version of table metadata is stored in a metadata folder under the table’
 
 Notes:
 
-1. The file system table scheme is implemented in [HadoopTableOperations](../../../javadoc/{{ icebergVersion }}/index.html?org/apache/iceberg/hadoop/HadoopTableOperations.html).
+1. The file system table scheme is implemented in [HadoopTableOperations](javadoc/latest/{{ icebergVersion }}/index.html?org/apache/iceberg/hadoop/HadoopTableOperations.html).
```
[the remark this answers is not in the document]
Comment (nastra) [reply format/spec.md:791]: even when removing `icebergVersion`, this constructs a URL that is relative to the current page: 
`spec/javadoc/latest/index.html?org/apache/iceberg/hadoop/HadoopTableOperations.html`

**Review by nastra — CHANGES_REQUESTED** [review]
javadoc links aren't working, but the other changes look good

**Comment by bitsondatadev** [comment]
lol I forgot this is from another directory higher than the docs...I know relative links are better in the long run but ugh they're a pain to debug in the beginning.

File: format/spec.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (bitsondatadev) [reply format/spec.md]: just gonna fix the image first

**Review by amogh-jahagirdar — APPROVED** [review]

**Comment by amogh-jahagirdar** [comment]
Discussed with @bitsondatadev right now we're aiming to get the image fixed first, that works for me locally.

**Pull request merged**